casepro/statistics/models.py

```diff
@@ -122,6 +124,30 @@ class Meta:
         abstract = True
 
 
+class BaseAverage(BaseCount):
+    """
+    Tracks total minute counts of different items (e.g. time since assigned ) in different scopes (e.g. org, user)
```

Comment:
The average isn't stored directly but computer from total / count. The wording could be better though.